providers/fab/src/airflow/providers/fab/auth_manager/dag_permissions.py

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
I would move this code either in `providers/fab/src/airflow/providers/fab/auth_manager/cli_commands/permissions_command.py` or in a new file in `providers/fab/src/airflow/providers/fab/auth_manager/cli_commands/`